src/main/java/uk/gov/pay/card/model/CardInformation.java

```diff
@@ -1,37 +1,31 @@
 package uk.gov.pay.card.model;
 
-import java.util.Collections;
-import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
 
 import static java.lang.String.format;
 
 public class CardInformation {
 
-    private String brand;
-    private CardType cardType;
-    private String label;
+    private final String brand;
+    private final CardType cardType;
+    private final String label;
     private Long min;
     private Long max;
-    private boolean corporate;
-    private PrepaidStatus prepaidStatus;
-
-    private static final Map<String, String> brandMapping;
-
-    static {
-        Map<String, String> brands = new HashMap<>();
-        brands.put("MC", "master-card");
-        brands.put("MCI DEBIT", "master-card");
-        brands.put("MCI CREDIT", "master-card");
-        brands.put("MAESTRO", "maestro");
-        brands.put("AMERICAN EXPRESS", "american-express");
-        brands.put("DINERS CLUB", "diners-club");
-        brands.put("VISA CREDIT", "visa");
-        brands.put("VISA DEBIT", "visa");
-        brands.put("ELECTRON", "visa");
-        brandMapping = Collections.unmodifiableMap(brands);
-    }
+    private final boolean corporate;
+    private final PrepaidStatus prepaidStatus;
+
+    private static final Map<String, String> brandMapping = Map.of(
```

Comment:
Forgot to say in my original review — could you change this to be `BRAND_MAPPING` since it’s a constant?